### PR TITLE
Added a mention to the Symfony Demo application

### PR DESCRIPTION
**Pull request by javiereguiluz**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes |
| Applies to | all |
| Fixed tickets | - |

### CODE REVIEW

File: book/installation.rst [hunk 1]
```diff
@@ -304,6 +304,30 @@ several minutes to complete.
 
 .. _installing-a-symfony2-distribution:
 
+Installing the Symfony Demo application
```
Comment (xabbuh) [inline book/installation.rst]: Application

Comment (javiereguiluz) [inline book/installation.rst]: Fixed. Thanks.

**Comment by xabbuh** [comment]
:+1: Nice addition Javier.

File: book/installation.rst [hunk 2]
```diff
@@ -304,6 +304,30 @@ several minutes to complete.
 
 .. _installing-a-symfony2-distribution:
```
Comment (WouterJ) [inline book/installation.rst]: this anchor should be moved to the below section

File: book/installation.rst [hunk 3]
```diff
@@ -304,6 +304,30 @@ several minutes to complete.
...
 
+Installing the Symfony Demo Application
+---------------------------------------
+
+The Symfony Demo application is a fully-functional application that shows in
+practice the recommended way to develop Symfony applications. The application
```
Comment (WouterJ) [inline book/installation.rst]: "in practice" feels a bit odd in this sentence. Maybe move it to the end of the sentence?

File: book/installation.rst (hunk 2 above)
Comment (javiereguiluz) [inline book/installation.rst]: Good catch!

File: book/installation.rst (hunk 3 above)
Comment (javiereguiluz) [inline book/installation.rst]: You are right. What if we remove "in practice" entirely? It feels redundant.

**Comment by weaverryan** [comment]
Awesome! The only question is - where else should we have links? Quick tour?

**Comment by harikt** [comment]
Awesome and thank you.

**Pull request merged**